pallets/parachain-staking/src/lib.rs

```diff
@@ -1946,7 +2005,10 @@ pub mod pallet {
 			let mut unstaking = <Unstaking<T>>::get(who);
 
 			ensure!(
-				unstaking.len() <= T::MaxUnstakeRequests::get(),
+				u32::try_from(unstaking.len())
```

Comment:
All the other parts `usize` is simply used `as u32`, and here it is not. Perhaps we should decide on which one to use?

Reply:
yeah converting with as is not that nice in case it's bigger than a u32. But since wasm is u32 only it should be fine. Felt fancy, removed it... f566feaf5fcbe8d4c32e3bd80ff735e22d9fcd7d